RELEASING.md

```diff
@@ -173,16 +173,22 @@ Only after the artifacts are available on Maven Central can you merge the pull r
 Otherwise, for major and minor releases your pull request should have added new Docker images, and you need to log into [Debezium's Docker Hub organization](https://hub.docker.com/r/debezium/) and add/update the build settings for each of the affected images.
 
 With every release the Docker image for PostgreSQL needs to be updated as well.
-First create a tag in the https://github.com/debezium/postgres-decoderbufs[postgres-decoderbufs] repository:
+First create a tag in the [postgres-decoderbufs](https://github.com/debezium/postgres-decoderbufs) repository:
 
     $ git tag v<%version%> && git push upstream v<%version%>
 
-Then update the Debezium version referenced in the https://github.com/debezium/docker-images/blob/master/postgres/9.6/Dockerfile#L22[postgres Docker file]
+Then update the Debezium version referenced in the [postgres Docker file](https://github.com/debezium/docker-images/blob/master/postgres/9.6/Dockerfile#L22)
 and push that commit which will cause the image to be re-published on Docker Hub automatically.
 
 When doing a new minor release (e.g. 0.7), log into Docker Hub and modify the "latest" tags to point to that new version.
```

Comment:
Ha, that sentence became obsolete with your change. I've removed it and amended your commit.